lib/iris/cube.py

```diff
@@ -4461,16 +4461,25 @@ def regrid(self, grid, scheme):
             target grid. The regridding schemes currently available
             in Iris are:
 
-                * :class:`iris.analysis.Linear`,
-                * :class:`iris.analysis.Nearest`, and
-                * :class:`iris.analysis.AreaWeighted`.
+                * :class:`iris.analysis.Linear`\*,
+                * :class:`iris.analysis.Nearest`\*,
+                * :class:`iris.analysis.AreaWeighted`\*,
+                * :class:`iris.analysis.UnstructuredNearest`,
+                * :class:`iris.analysis.PointInCell`,
+                * :class:`iris.experimental.regrid.ProjectedUnstructuredLinear`,
+                * :class:`iris.experimental.regrid.ProjectedUnstructuredNearest`.
```

Comment:
I found several more regridding schemes and added them here. Does that look right, or were they missing for a reason?

Reply:
I think this is out of data.  But also, we didn't want to commit to the experimental ones.
So maybe it should omit those, and amend to say "The regridding schemes in Iris currently *include*:" ...

Reply:
Also.. if we extend this then the next section that you added "These regridding schemes support lazy regridding..." needs to change, as *not* all these schemes support it.

Reply:
I removed the experimental regridders and changed the section in the user guide so it explicitly mentions which regridding schemes are lazy.

Reply:
I did not add a section or mention of the UnstructeredNearest or PointInCell schemes to the user guide, because I would not feel qualified to write such a section and I also feel that it would be beyond the scope of this pull request.